core/src/main/java/org/elasticsearch/common/geo/GeoDistance.java

```diff
@@ -41,7 +41,9 @@
 public enum GeoDistance implements Writeable {
     /**
      * Calculates distance as points on a plane. Faster, but less accurate than {@link #ARC}.
+     * @deprecated use {@link GeoUtils#planeDistance}
      */
+    @Deprecated
```

Comment:
Forgot to mention the GeoDistance class is coupled with the GeoDistance query parser. That's why I started w/ deprecation. I plan to completely remove the GeoDistance class but would rather do it in a few small PRs than one large one. Any issue keeping it in this one and I'll remove it in the next PR?